source/css/sass/mixins/_typography.scss

```diff
@@ -66,3 +71,40 @@
   max-height: 1em;
   vertical-align: middle;
 }
+
+@mixin _label-typeg($uppercase: true) {
+  font-family: $font-secondary;
+  font-weight: normal;
+  @include font-size-and-vertical-height(11);
+  letter-spacing: 0.5px;
+  @if $uppercase {
+    text-transform: uppercase;
+  }
+}
+
+@mixin label-content-typeg($color:$color-text-secondary, $uppercase: true) {
+  @include _label-typeg($uppercase);
+  color: $color;
+}
+
+@mixin label-subject-typeg() {
+  @include _label-typeg();
+  color: $color-primary-normal;
+}
+
+@mixin font-size($size_in_px) {
+  font-size: #{$size_in_px}px;
+  font-size: get-rem-from-px($size_in_px) * 1rem;
+}
+
+@function get-rem-from-px($size_in_px) {
+  @return $size_in_px/$font-size-base-in-px;
```

Comment:
`$font-size-base-in-px` has been renamed to `$font-size-base_in_px` (derived from https://github.com/libero/pattern-library/blob/master/libero-config/config--libero-default.js#L60). If `$font-size-base-in-px` is building for you, your config may be out of date. (Update by running the gulp task `distributeSharedConfig`).